wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/AstNodeToStaticString.scala

```diff
@@ -0,0 +1,16 @@
+package wdl.transforms.base.ast2wdlom
+
+import common.transforms.CheckedAtoB
+import common.validation.Checked._
+import wdl.model.draft3.elements._
+
+object AstNodeToStaticString {
+  def astNodeToStaticStringElement(): CheckedAtoB[GenericAstNode, StaticString] = CheckedAtoB.fromCheck("convert AstNode to StaticString") {
+    case a: GenericAst if a.getName == "StaticString" =>
+      val value = if (a.getAttributes.contains("value")) {
+        a.getAttributeAs[String]("value")
+      } else "".validNelCheck
```

Comment:
I'm guessing this is the key piece of logic for this PR - a `StaticString` AST node that represents an empty string doesn't directly give you `""`, you have to infer it by the absence of the `"value"` attribute?

(Might be nice to comment if accurate)